### ISSUE TITLE
Cannot download backup -> Error: "Backup File Not Found!"

### ISSUE THREAD

**Issue by petrhejna** [Ride-The-Lightning/RTL#536]
**Describe the bug**
When trying to download backup I got this error:
```
Reading File Failed!
Error Code
-2
Error Message
Backup File Not Found!
API URL
./api/conf/file?channel=ALL
```

**To Reproduce**
1. Go to 'Lightning -> Backup'
2. Click on 'Backup All'
3. It will show success message.
4. File `channel-all.bak` is correctly created in displayed location.
5. Clicking on "Download Backup" produces mentioned error 

**Screenshots**
![image](https://user-images.githubusercontent.com/383295/98528974-56f3b280-227d-11eb-92de-4da2b715a134.png)

**Your environment**
* Version of `RTL`: `0.9.3`
* Version of `lnd`: `lnd version 0.11.1-beta commit=v0.11.1-beta`
* Version of `btcd`, `bitcoind`, or other backend: `Bitcoin Core version v0.20.0`
* Browser & browser version: `Mozilla Firefox 82.0`
* Operating system (`uname -a` on *Nix): `inux raspberrypi 4.19.118-v7l+ #1311 SMP Mon Apr 27 14:26:42 BST 2020 armv7l GNU/Linux`

**Comment by arvchristos** [comment]
Adding to this topic, this is the error that I get when I try to download channel backup:

```
ERROR: Conf(207) => Reading File Failed!{"errno":-2,"code":"ENOENT","syscall":"open","path":"/rtl_chan_backup/channel-ALL.bak"}
```

When checking the rtl_chan_backup directory, I find `channel-all.bak` but not `channel-ALL.bak`

**Comment by ShahanaFarooqui** [comment]
Thanks for reporting the bug. It was due to case sensitive file name. The fix has been pushed to branch 0.10.0 and will be released within 1-2 weeks.
